website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -38,24 +38,35 @@ You can install the dbt Cloud CLI on the command line by using one of these meth
 Before you begin, make sure you have [Homebrew installed](http://brew.sh/) in your code editor or command line terminal. Refer to the [FAQs](#faqs) if your operating system runs into path conflicts.
 
 
-1. Run the following command to verify that there is no conflict with a dbt Core installation on your system:
+1. Run the following command to verify that you don't already have dbt Core installed:
    
   ```bash
   which dbt
   ```
-   - This should return a `dbt not found`. If the dbt help text appears, use `pip uninstall dbt` to deactivate dbt Core from your machine.
+   - This should return a `dbt not found`. If the dbt help text appears, use `pip uninstall dbt` to remove dbt Core from your machine. <br />
   
 2. Install the dbt Cloud CLI with Homebrew: 
 
-  ```bash
-  brew untap dbt-labs/dbt
-  brew tap dbt-labs/dbt-cli
-  brew install dbt
-  ```
-
-3. Verify the installation by running `dbt --help` from the command line. If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a version of dbt globally installed.
+   - First, remove a tap, which is like a separate repository for packages, from Homebrew. This stops Homebrew from installing packages from that repository:
+      ```bash
+      brew untap dbt-labs/dbt
+   -  Then run `brew tap` to add and install the dbt Cloud CLI as a package:
+      ```bash
+      brew tap dbt-labs/dbt-cli
+      ```
+   - Lastly, install the dbt Cloud CLI with Homebrew:
+      ```bash
+      brew install dbt
+      ```
+
+3. Verify the installation by running `dbt --help` from the command line. 
+   * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a global dbt version installed.
```

Comment:
Should we drop some sample text of what the expected output is?

Reply:
i thought about it and opted to leave it out because `dbt --help` returns a pretty long output (basically it gives users all the flags/commands they can use) so i decided to leave it out and thought that the 'help text...' would be clear enough

Reply:
"`dbt --help` returns a pretty long output.." is the text that indicates it is successfully running for cloud going to be immediately apparent in this output or will they have to dig for it? Maybe including just a sample of what to look for so they know 100% it was successful?

Reply:
I think I'm maybe just confused by what this sentence is saying. Are we saying that the ability to run `dbt --help` is enough to confirm installation, or are they supposed to see something that specifically states it's the Cloud CLI and not the Core CLI?  If it's the former, maybe we can clarify it by saying something like: "If the installation was successful, executing `dbt --help` will produce a list of commands available to run."

Reply:
ok maybe this clarifies it further?

```
Verify your installation by running dbt --help in the command line. If you see the following output, your installation is correct

The dbt Cloud CLI - an ELT tool for running SQL transformations and data models in dbt Cloud...

If you don't see this output, check that you've deactivated pyenv or venv and don't have a global dbt version installed.

Note that you no longer need to run the dbt deps command when your environment starts. This step was previously required during initialization. However, you should still run dbt deps if you make any changes to your packages.yml file.
```

![Screenshot 2023-10-25 at 10 39 43](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/7f3f76e9-5b00-421f-92ae-7394cc78e86f)